src/de/jost_net/JVerein/server/DDLTool/Updates/Update0479.java

```diff
@@ -92,7 +92,8 @@ public void run() throws ApplicationException
             break;
           case Types.CHAR:
           case Types.VARCHAR:
-            value = result.getString(i).replace("\n", "\\n");
+            value = result.getString(i).replace("\n", "\\n").replace("\\",
```

Comment:
Sind in dem String, welcher aus der Datenbank gelesen wird, tatsächlich double-backslashes drin?